drizzlepac/alignimages.py

```diff
@@ -873,13 +846,12 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
         chip_num = item.meta['chip']
 
         # Build fit_status_dict entry
-        dict_key = "{},{}".format(image_name, chip_num)
-        # Initialize dictionary entry for current image/chip
-        fit_status_dict[dict_key] = {'valid': False,
-                                     'max_rms': max_rms_val,
-                                     'num_matches': num_xmatches,
-                                     'compromised': False,
-                                     'reason': ""}
+        dictKey = "{},{}".format(image_name, chip_num)
```

Comment:
Variable should not be camelcase. Rename 'dictKey' to dict_key and find/replace the remaining instances.

Reply:
Resolved in 51e3cea